notebooks/dataAccess/siphon-catalog-basics.ipynb

```diff
@@ -0,0 +1,384 @@
+    "If you change the last part of the URL from `.html` to `.xml` (that is, <https://thredds.ucar.edu/thredds/catalog/catalog.html>), you will see the actual THREDDS Catalog in your browser, which looks similar to this:\n",
+    "\n",
+    "~~~xml\n",
+    "<catalog xmlns=\"http://www.unidata.ucar.edu/namespaces/thredds/InvCatalog/v1.0\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"   name=\"Unidata THREDDS Data Server\" version=\"1.0.1\">\n",
+    "  <dataset name=\"Realtime data from IDD\">\n",
+    "    <catalogRef xlink:href=\"idd/forecastModels.xml\" xlink:title=\"Forecast Model Data\" name=\"\"/>\n",
+    "    <catalogRef xlink:href=\"idd/forecastProdsAndAna.xml\" xlink:title=\"Forecast Products and Analyses\" name=\"\"/>\n",
+    "    <catalogRef xlink:href=\"idd/obsData.xml\" xlink:title=\"Observation Data\" name=\"\"/>\n",
+    "    <catalogRef xlink:href=\"idd/radars.xml\" xlink:title=\"Radar Data\" name=\"\"/>\n",
+    "    <catalogRef xlink:href=\"idd/satellite.xml\" xlink:title=\"Satellite Data\" name=\"\"/>\n",
+    "  </dataset>\n",
+    "  <dataset name=\"Other Unidata Data\">\n",
+    "    <catalogRef xlink:href=\"casestudies/catalog.xml\" xlink:title=\"Unidata case studies\" name=\"\"/>\n",
+    "  </dataset>\n",
+    "</catalog>\n",
+    "~~~\n",
+    "\n",
+    "This catalog tells us that there are other catalogs containing data from forecast models, radar data, satellite data, etc.\n",
+    "In this sense, a catalog can point to other catalogs, creating a tree-like structure in which the datasets are organized.\n",
+    "This will vary from server to server, as needs vary across organizations and groups.\n",
+    "\n",
+    "We can use Siphon to read in this remote catalog programmatically, without the need for a web browser:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "catalog = TDSCatalog('http://thredds.ucar.edu/thredds/catalog/catalog.xml')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have read in the THREDDS Catalog from the THREDDS Data Server, we can investigate what information it holds.\n",
+    "A list of the names of other catalogs it points to is contained within the `catalog_refs` instance attribute, and can be access as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "catalog.catalog_refs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are more things you can do once you have read a THREDDS Catalog using siphon, but for now we'll leave it at this.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Note: Not all TDS catalogs are intended to be browsed directly:</b> Occasionally, the TDS is used purely as \"middleware\", and the catalogs are not setup for users to easily browse directly.</div>\n",
```

Comment:
The NA-CORDEX example is not showing up in the "Note" box, which makes it look like part of the tutorial and not part of an aside. Can you group the note and the example together?